client/my-sites/marketing/do-it-for-me/site-info-collection.tsx

```diff
@@ -1,72 +1,76 @@
-import { Button } from '@automattic/components';
-import { useShoppingCart } from '@automattic/shopping-cart';
+import config from '@automattic/calypso-config';
 import styled from '@emotion/styled';
+import { Widget } from '@typeform/embed-react';
+import { useTranslate } from 'i18n-calypso';
 import page from 'page';
 import React, { useState } from 'react';
 import { useSelector } from 'react-redux';
 import ActionPanel from 'calypso/components/action-panel';
 import ActionPanelBody from 'calypso/components/action-panel/body';
-import ActionPanelFooter from 'calypso/components/action-panel/footer';
+import CardHeading from 'calypso/components/card-heading';
 import FormattedHeader from 'calypso/components/formatted-header';
-import { fillInSingleCartItemAttributes } from 'calypso/lib/cart-values';
+import Spinner from 'calypso/components/spinner';
 import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
-import { getProductsList } from 'calypso/state/products-list/selectors';
+import { getCurrentUserName } from 'calypso/state/current-user/selectors';
 import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
 
-const TemporaryTypeformContainer = styled.div`
-	width: 95%;
-	height: 18em;
-	padding: 1em 0 0 1em;
-	background-color: #eee;
-	font-size: 2em;
-	color: #b8b8b8;
+function getTypeformId(): string {
+	return config< string >( 'difm_typeform_id' );
+}
+
+const LoadingContainer = styled.div`
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	height: 500px;
+	justify-content: center;
 `;
 
-function SiteInformationCollection() {
+const SiteInformationCollection = React.memo( () => {
```

Comment:
Not sure if this is the right way to go here since we should not rely on memo to fix issues in functionality. Maybe we can look at this later though.

From the docs. : https://reactjs.org/docs/react-api.html#reactmemo
> _This method only exists as a performance optimization. Do not rely on it to “prevent” a render, as this can lead to bugs_

Reply:
You're right, I added this while testing and then missed removing this.